### DISCUSSION TITLE
FEPX   0% tests passed, 30 tests failed out of 30

### DISCUSSION THREAD

**Discussion by zengyan0112** [neperfepx/FEPX#103] [Q&A] [answered]
0% tests passed, 30 tests failed out of 30

Total Test time (real) =   0.10 sec

The following tests FAILED:
          1 - 01_bcc (Failed)
          2 - 02_fcc (Failed)
          3 - 03_hcp (Failed)
          4 - 04_bct (Failed)
          5 - 05_dual_bcc_fcc (Failed)
          6 - 06_bcc_sat_evo (Failed)
          7 - 07_fcc_sat_evo (Failed)
          8 - 08_hcp_sat_evo (Failed)
          9 - 09_bct_sat_evo (Failed)
         10 - 10_hcp_aniso_m (Failed)
         11 - 11_bct_aniso_m (Failed)
         12 - 12_bcc_aniso_hard (Failed)
         13 - 13_fcc_aniso_hard (Failed)
         14 - 14_hcp_aniso_hard (Failed)
         15 - 15_bct_aniso_hard (Failed)
         16 - 16_bcc_cyclic_hard (Failed)
         17 - 17_fcc_cyclic_hard (Failed)
         18 - 18_hcp_cyclic_hard (Failed)
         19 - 19_bct_cyclic_hard (Failed)
         20 - 20_triaxial_csr (Failed)
         21 - 21_triaxial_clr (Failed)
         22 - 22_oris_rod_passive (Failed)
         23 - 23_oris_bunge_active (Failed)
         24 - 24_oris_bunge_passive (Failed)
         25 - 25_oris_kocks_active (Failed)
         26 - 26_oris_kocks_passive (Failed)
         27 - 27_oris_axisangle_active (Failed)
         28 - 28_oris_axisangle_passive (Failed)
         29 - 29_oris_quat_active (Failed)
         30 - 30_oris_quat_passive (Failed)
Errors while running CTest
make: *** [Makefile:107: test] Error 8

**Comment by mkasemer** [comment, marked as answer]
Have you built FEPX properly prior to running these tests? Are you running on the exact same setup as the tests were generated?

**Reply by zengyan0112** [reply]
> Have you built FEPX properly prior to running these tests? Are you running on the exact same setup as the tests were generated?

Thank you for your reply.
I think I have built FEPX properly, and here is the built results.
![image](https://user-images.githubusercontent.com/94596383/217973632-9235072c-58a6-4eaf-95da-7b5b8964a5f8.png)

**Reply by mkasemer** [reply]
Are you running the tests on the exact same setup as the tests were generated, as described in the manual? I suspect not, hence the universal failures.

Please consult the manual, and it will explain why these are likely erroneous failures.

**Reply by zengyan0112** [reply]
> Are you running the tests on the exact same setup as the tests were generated, as described in the manual? I suspect not, hence the universal failures.
> 
> Please consult the manual, and it will explain why these are likely erroneous failures.

Thank you for response, I can't find the  explain why these are likely erroneous failures. Because I have try the changing of testing mode.

**Reply by rquey** [reply]
@zengyan0112 Please stick to the latest version.

Try to run the tests in `Minimal` mode as described in the documentation:
```
cmake -DBUILD_TESTING_MODE=Minimal ..
make
ctest
```

If they pass, you should be safe.  If they don't, it may be an MPI issue - go to one of the test folders (e.g. `tests/01_bcc`) and run `mpirun -np 2 fepx` there.

**Reply by rquey** [reply]
Please simply copy the text instead of inserting images.

It seems that you have write permission issues in this folder (which does not depend on FEPX). This needs to be fixed.

**Reply by zengyan0112** [reply]
> Please simply copy the text instead of inserting images.
> 
> It seems that you have write permission issues in this folder (which does not depend on FEPX). This needs to be fixed.

Thank you very much, I will have a try and delete the image.

**Reply by zengyan0112** [reply]
> Please simply copy the text instead of inserting images.
> 
> It seems that you have write permission issues in this folder (which does not depend on FEPX). This needs to be fixed.

Thank you very much for your sincere and professional help, I have been successful：
`100% tests passed, 0 tests failed out of 30
Total Test time (real) =  43.75 sec`
